editoast/src/client/infra_commands.rs

```diff
@@ -120,32 +123,42 @@ pub async fn import_railjson(
         .last_railjson_version();
     let railjson: RailJson = serde_json::from_reader(BufReader::new(railjson_file))?;
 
-    println!("🍞 Importing infra {infra_name}");
+    if !args.quite {
+        println!("🍞 Importing infra {infra_name}");
+    }
```

Comment:
Usually for CLI tools `stderr` is used for user-facing messages, and `stdout` is used for machine-readable content. This would remove the need for a specific flag.